static/js/ui-utils.js

```diff
@@ -3,119 +3,7 @@
-    $('#alarm-urgenthigh-browser').prop('checked', json.alarmUrgentHigh).next().text('Urgent High Alarm' + appendThresholdValue(app.thresholds.bg_high));
-    $('#alarm-high-browser').prop('checked', json.alarmHigh).next().text('High Alarm' + appendThresholdValue(app.thresholds.bg_target_top));
-    $('#alarm-low-browser').prop('checked', json.alarmLow).next().text('Low Alarm' + appendThresholdValue(app.thresholds.bg_target_bottom));
-    $('#alarm-urgentlow-browser').prop('checked', json.alarmUrgentLow).next().text('Urgent Low Alarm' + appendThresholdValue(app.thresholds.bg_low));
-    $('#alarm-timeagowarn-browser').prop('checked', json.alarmTimeAgoWarn);
-    $('#alarm-timeagowarnmins-browser').val(json.alarmTimeAgoWarnMins);
-    $('#alarm-timeagourgent-browser').prop('checked', json.alarmTimeAgoUrgent);
-    $('#alarm-timeagourgentmins-browser').val(json.alarmTimeAgoUrgentMins);
-
-    json.nightMode = setDefault(json.nightMode, app.defaults.nightMode);
-    $('#nightmode-browser').prop('checked', json.nightMode);
-
-    if (rawBGsEnabled()) {
-      $('#show-rawbg-option').show();
-      json.showRawbg = setDefault(json.showRawbg, app.defaults.showRawbg);
-      $('#show-rawbg-' + json.showRawbg).prop('checked', true);
-    } else {
-      json.showRawbg = 'never';
-      $('#show-rawbg-option').hide();
-    }
-
-    json.customTitle = setDefault(json.customTitle, app.defaults.customTitle);
-    $('h1.customTitle').text(json.customTitle);
-    $('input#customTitle').prop('value', json.customTitle);
-
-    json.theme = setDefault(json.theme, app.defaults.theme);
-    if (json.theme === 'colors') {
-      $('#theme-colors-browser').prop('checked', true);
-    } else {
-      $('#theme-default-browser').prop('checked', true);
-    }
-
-    json.timeFormat = setDefault(json.timeFormat, app.defaults.timeFormat);
-
-    if (json.timeFormat === '24') {
-      $('#24-browser').prop('checked', true);
-    } else {
-      $('#12-browser').prop('checked', true);
-    }
-
-    json.showPlugins = setDefault(json.showPlugins, app.defaults.showPlugins || Nightscout.plugins.enabledPluginNames());
-    var showPluginsSettings = $('#show-plugins');
-    Nightscout.plugins.eachEnabledPlugin(function each(plugin) {
-      if (Nightscout.plugins.specialPlugins.indexOf(plugin.name) > -1) {
-        //ignore these, they are always on for now
-      } else {
-        var id = 'plugin-' + plugin.name;
-        var dd = $('<dd><input type="checkbox" id="' + id + '" value="' + plugin.name + '"/><label for="' + id + '">' + (plugin.label || plugin.name) + '</label></dd>');
-        showPluginsSettings.append(dd);
-        dd.find('input').prop('checked', json.showPlugins.indexOf(plugin.name) > -1);
-      }
-    });
-
-
-  } catch(err) {
-    console.error(err);
-    showLocalstorageError();
-  }
-
-  return json;
+  return serverSettings.enabledOptions && serverSettings.enabledOptions.indexOf('rawbg') > -1;
 }
 
 function setDefault(variable, defaultValue) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['setDefault' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/799298463/issues/source?bid=2452379&fileBranchId=2725007#l9)